src/bootstrap/src/core/config/config.rs

```diff
@@ -1440,14 +1451,24 @@ impl Config {
             config.out = crate::utils::helpers::absolute(&config.out);
         }
 
+        // Hacky way to determine the executable suffix for the build target. We cannot use
+        // std::env::consts::EXE_SUFFIX as the build target might not be the target bootstrap was
+        // compiled with.
+        let initial_exe_suffix = if config.build.triple.contains("windows") { ".exe" } else { "" };
```

Comment:
You can use `utils::helpers::exe` function.

ref:
https://github.com/rust-lang/rust/blob/b4b2643c111714b541db1c18c178ad0ce654238b/src/bootstrap/src/core/config/config.rs#L1941

Reply:
TIL about that helper! Updated.